extensions/twitch/general.py

```diff
@@ -149,6 +168,26 @@ async def http(self, ctx: commands.Context, *, code: str) -> None:
     async def keyboard2(self, ctx: commands.Context) -> None:
         await ctx.reply("Logitech MX Mechanical Mini Wireless Illuminated Keyboard, Clicky Switches")
 
+    @commands.command()
+    async def count(self, ctx: commands.Context, mbti_type: str) -> None:
+        data: dict[str, int] = self.bot.dbot.mbti_count()
+        total: int | None = data.get(mbti_type.upper())
+        if total is None:
+            await ctx.reply(f"There are no {mbti_type} types in the server!")
+            return
+        await ctx.reply(f"There are {total} {mbti_type} types in the server!")
+
+    @routines.routine(seconds=1)
+    async def midnight(self) -> None:
+        channel: twitchio.Channel | None = self.bot.get_channel("timeenjoyed")
+        assert channel is not None
+        for timezone in self.timezones:
+            tz = zoneinfo.ZoneInfo(timezone)
+            current = datetime.now(tz)
+            if current.hour == 23 and current.minute == 47 and current.second == 0:
+                logger.info(f"it is {current} in {tz}")
```

Comment:
This should probably be a `debug` log entry?

Reply:
And maybe a little more descriptive? E.g.

`'The "midnight" Routine has detected it is midnight in ...'`